ktx-completion.sh

```diff
@@ -17,6 +17,6 @@
 KUBECONFIG_DIR=${KUBECONFIG_DIR:-"${HOME}/.kube/"}
 _ktx() {
        local cur=${COMP_WORDS[COMP_CWORD]}
-       COMPREPLY=( $(compgen -W "$(ls ${KUBECONFIG_DIR})" -- $cur) )
+       COMPREPLY=( $(compgen -W "$(find ${KUBECONFIG_DIR} -maxdepth 1 -type f -exec basename {} \;)" -- $cur) )
```

Comment:
How would you feel about extracting this `find` out to a function? I agreed with your other point about testing the files to only show kubeconfig files so I expect more logic in future.

Up to you, happy to merge, just lmk.